src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -29,22 +29,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         Private Function FindTriviaToLeft(nodeOrToken As SyntaxNodeOrToken, position As Integer) As SyntaxTrivia
+recurse:
             For Each child In nodeOrToken.ChildNodesAndTokens().Reverse()
-                If child.FullSpan.Start < position AndAlso position <= child.FullSpan.End Then
+                If (child.FullSpan.Start < position) AndAlso (position <= child.FullSpan.End) Then
                     If child.IsNode Then
-                        Return FindTriviaToLeft(child, position)
+                        nodeOrToken = child
+                        GoTo recurse
                     Else
                         Dim triviaList = child.GetLeadingTrivia().Concat(child.GetTrailingTrivia()).Reverse()
```

Comment:
Unrelated to your change, it would also be nice to get rid of this unnecessary allocation.  There can be separate iterations of trailing and leading trivia.